client/src/pages/DirectoryPage/DirectoryGroups.js

```diff
@@ -14,223 +14,196 @@ import picture1 from "../../assets/images/card_small1.png";
-            selectId={0}
-          />{" "}
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />{" "}
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
-          />
-          <Pill
-            className="Pill"
-            text={"Traveling"}
-            size={"wide"}
-            color={"pink"}
-            icon={"white"}
-            onClick={pillClick}
-            selectId={0}
+  componentDidMount() {
+    axios.get(`${BASE_URL}/groups`)
+      .then(response => {
+        this.setState({ 
+          groups: response.data
+        })
+      })
+      .catch(error => {
+        console.log('Error while fetching and parsing API request', error);
+      })
+  }
+
+  render() {
+    console.log(this.state)
+    const groups = this.state.groups ? this.state.groups : [];
+    return (
+      <div className="DirectoryPage">
+        <NavBar page={0} profileImage={people} />
+        <Banner background={Banner__Directory}>
+          <h3 style={{ margin: 0 }}>Directory: My Groups</h3>
+          <InputTextField
+            type="text"
+            variation="wide"
+            name="search directory"
+            placeholder="Search Groups"
           />
-        </div>{" "}
-        <a className="seeMore-Button">See More</a>
-        <div>
-          <header className="CommunityPage__header">Groups in WASHINGTON, DC:</header>
+        </Banner>
+        <div className="secondNav">
+          <a className="secondNav" href="./people">
+            <Button text="My Friends" color="outline" size="long" />
+          </a>{" "}
+          <a className="secondNav" href="./groups">
+            <Button text="My Groups" color="purple" size="long" />
+          </a>
         </div>
-        <div className="CommunityPage__groups-in-city-container">
-          <div className="CommunityPage__groupCard-single">
-            <GroupCard
-              name="DIY with your kids"
-              members="98 Members"
-              location="Mixed Locations"
-              picture={picture1}
-            />
+
+        <div className="CommunityPage_SortByButton">
+          <div className="CommunityPage_SortByText">Sort By: Location </div>
+        </div>
```

Comment:
This is a little strange - why is the class `CommunityPage_SortByButton` when there's no button here?